Brewfile

```diff
@@ -32,6 +32,7 @@ brew 'git'
 # PHP and tools
 brew 'homebrew/php/php70', args: ['with-gmp']
 brew 'homebrew/php/php70-xdebug'
+brew 'homebrew/php/php-cs-fixer'
```

Comment:
Could you move this to global Composer installation? We already have it there, but it's an outdated version. They don't recommend the Brew installation and the version management is a pain with Brew.

Reply:
My comment is in line with #2 which I just noticed again.